dvc/command/plot.py

```diff
@@ -0,0 +1,225 @@
+import argparse
+import logging
+import os
+
+from dvc.command.base import append_doc_link, CmdBase, fix_subparsers
+from dvc.exceptions import DvcException
+from dvc.repo.plot.data import WORKSPACE_REVISION_NAME
+
+logger = logging.getLogger(__name__)
+
+
+class CmdPLot(CmdBase):
+    def _revisions(self):
+        raise NotImplementedError
+
+    def _result_file(self):
+        if self.args.result:
+            return self.args.result
+
+        extension = self._result_extension()
+        base = self._result_basename()
+
+        result_file = base + extension
+        if os.path.exists(result_file):
+            raise DvcException(
+                "Cannot create '{}': file already exists, use -r to redefine "
```

Comment:
```suggestion
                "Cannot create '{}': file already exists, use `-r` to redefine "
```
Why not -f though? Like in `dvc add/run`